### DISCUSSION TITLE
formatting for query `scm`files

### DISCUSSION THREAD

**Discussion by molleweide** [nvim-treesitter/nvim-treesitter#2875] [Q&A]
Hi guys,

I was wondering if there are any cool tools for formatting and making queries look good?
Is there anything in particular that I should check out? I haven't really delved into this at all so I am just throwing the question out there. 

Is anyone using eg. Lisp formatting for queries? I am not that familiar with lisps so maybe it doesn't make any sense.
However, now that I am starting to get a grasp of how to write queries, it is an awful lot of fun!

**Comment by kyazdani42** [comment]
something like parinfer ?

**Reply by molleweide** [reply]
I will check it out and se what it is. I am not sure what I am asking for exactly :P

**Comment by fsouza** [comment]
Do you mean like a code formatter? I think there's one for Scheme, but I haven't used it myself.

edit: link https://github.com/masukomi/scheme-format

**Reply by molleweide** [reply]
Cool thanks again man! I will check it out.
